content/codespaces/codespaces-reference/about-billing-for-codespaces.md

```diff
@@ -6,6 +6,7 @@ versions:
   free-pro-team: '*'
```

Comment:
https://youtube.com/c/SinEstresOrrantia

https://distrokid.com/hyperfollow/sinestres/not-another-broken-heart

https://distrokid.com/hyperfollow/sinestres/hablan-mal-de-mi

Check out Omar Orrantia’s profile on LinkedIn https://www.linkedin.com/in/omar-orrantia-13032999

https://distrokid.com/hyperfollow/sinestres/do-you-like-me	


```suggestion
  - /github/developing-online-with-github-codespaces/about-github-codespaces
```
	for ( i = 0 to i = input.length-1 ) {

				int temp = input[ i ]
				boolean hasMoved = false
				boolean notPlaced = true

				for ( j = i-1 to j = 0 ) {

						if ( input[ j ] < temp ) {
								//  Shift numbers till you find it’s right location
								input[ j+1 ] = input[ j ]
								hasMoved = true
						} else {
								input[ j+1 ] = temp
								notPlaced = false
								break;
						}

				}

				// Edge case
				if( hasMoved && notPlaced ) 
						input[0] = temp 

		}